site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -149,3 +172,52 @@ wget https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch
 chmod o+r gameserver.yaml
 docker run --network=host --rm -v $(pwd)/gameserver.yaml:/tmp/gameserver.yaml us-docker.pkg.dev/agones-images/release/agones-sdk:{{<release-version>}} --local -f /tmp/gameserver.yaml
 ```
+
+## Running locally using "out of cluster" mode
```

Comment:
Just a heads up, our docs publish on merge, so if you have new features that aren't in the current release, please wrap them in a `feature` shortcode like so: https://agones.dev/site/docs/contribute/

Reply:
Added `publishDate` to new page. Wrapped links with `{{% feature publishVersion="1.34.0" %}}`.